### PR TITLE
Webapp take tracking strategy from env

### LINKED ISSUE #406: Webapp to take TRACKING_STRATEGY into account
## Tell us about the problem you're trying to solve
Webapp has segment always enabled

## Describe the solution you’d like
Webapp should take TRACKING_STRATEGY into account and only use segment if configured so.

### PR DESCRIPTION
**Pull request by jamakase**
## What
This solution replaces all __SERVER_TRACKING_STRATEGY__ occurrences in code to env variable passed to nginx process. 

## How
This is achieved by using next Nginx features which are prebuilt in docker image:
- envsubst : replaces env variables in `template/nginx.conf.template` and places file to `/etc/nginx/conf.d`
- ngx_http_sub_module : replaces word occurrence in index.html before returning to client.

### CODE REVIEW

**Review by michel-tricot — APPROVED** [review]

File: airbyte-webapp/Dockerfile [hunk 1]
```diff
@@ -3,4 +3,4 @@ FROM nginx:1.19-alpine as webapp
 EXPOSE 80
 
 COPY build /usr/share/nginx/html
-COPY nginx/default.conf /etc/nginx/conf.d/default.conf
+COPY nginx/default.conf.template /etc/nginx/templates/default.conf.template
```
Comment (michel-tricot) [inline airbyte-webapp/Dockerfile:6]: why did you need to rename it?

Comment (Jamakase) [reply airbyte-webapp/Dockerfile:6]: It is not a config itself anymore, it is a template.

According to docs of nginx Dockerfile 
```By default, this function reads template files in /etc/nginx/templates/*.template and outputs the result of executing envsubst to /etc/nginx/conf.d.```
So `envsubst` will replace `$TRACKING_STRATEGY` in the template with env variable and make proper nginx config.

Comment (michel-tricot) [reply airbyte-webapp/Dockerfile:6]: Got it.
